### PR TITLE
Fix potential error in batch runner

### PR DESCRIPTION
**Pull request by myarjunar**

### CODE REVIEW

**Comment by timlinux** [comment]
LGTM

File: safe/gui/tools/batch/batch_dialog.py [hunk 1]
```diff
@@ -305,6 +313,7 @@ def prepare_task(self, items):
             hazard = self.define_layer(hazard_path)
             if not hazard:
                 status = False
+                message = 'Unable to find %s. ' % hazard_path
```
Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: `tr` ?

File: safe/gui/tools/batch/batch_dialog.py [hunk 2]
```diff
@@ -316,6 +325,7 @@ def prepare_task(self, items):
             exposure = self.define_layer(exposure_path)
             if not exposure:
                 status = False
+                message += '\nUnable to find %s. ' % exposure_path
```
Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: `tr` ?

**Comment by timlinux** [comment]
LGTM after translations added

File: safe/gui/tools/batch/batch_dialog.py (hunk 1 above)
Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: 'Unable to find {hazard_path}. '.format(hazard_path=hazard_path)

Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: It's better when we do translations, we can know the context in Transifex

File: safe/gui/tools/batch/batch_dialog.py (hunk 2 above)
Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: `'Unable to find {exposure_path}. '.format(exposure_path=hazard_path)`

**Pull request merged**